BrenKt-TRANS-SK1153-WEB-Domain-DMARC (#5679)
ovh/docs: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md

---
title: Configurar um registo DMARC no seu domínio
excerpt: Saiba como o DMARC funciona e como configurá-lo para o seu serviço de e-mail
updated: 2023-12-13
---

> [!primary]
> Esta tradução foi automaticamente gerada pelo nosso parceiro SYSTRAN. Em certos casos, poderão ocorrer formulações imprecisas, como por exemplo nomes de botões ou detalhes técnicos. Recomendamos que consulte a versão inglesa ou francesa do manual, caso tenha alguma dúvida. Se nos quiser ajudar a melhorar esta tradução, clique em "Contribuir" nesta página.
>

## Objetivo

O registo **D**omain-based **M**essage **A**uthentication, **R**eporting, and **C**onformance (DMARC) é um mecanismo de segurança de e-mail. Baseia-se no resultado das verificações [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim).

**Saiba como o DMARC funciona e como configurá-lo para o seu serviço de e-mail.**

> [!warning]
>
> A OVHcloud oferece serviços cuja configuração, gestão e responsabilidade são da sua responsabilidade. Assim, deverá certificar-se de que estes funcionam corretamente.
> 

> > Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/directory/). Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção [« Vá mais longe »](#go-further) deste tutorial.

Reviewer: ```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/directory/). Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção [Quer saber mais?](#go-further) deste tutorial.
```

>

> > Além disso, a implementação de um registo **DMARC**, independentemente do [serviço de e-mail OVHcloud](https://www.ovhcloud.com/fr-ca/emails/), está atualmente indisponível. Este guia diz **unicamente** respeito aos clientes que possuem uma zona DNS ativa na OVHcloud para o seu nome de domínio associado a uma **solução de e-mail externa**.

Reviewer: wrong link

- **Subdomínio**: Esta entrada deve **obrigatoriamente começar por** `_dmarc`. Se você está aplicando seu DMARC ao conjunto do domínio, não insira nada além de `_dmarc` nesta caixa. Se definir o DMARC para um subdomínio associado ao seu domínio principal, adicione o subdomínio após `_dmarc`. Por exemplo, se um utilizador tiver de aplicar o DMARC a um subdomínio *subdomain.mydomain.ovh*, deverá introduzir `_dmarc.subdomain` na casa « subdomínio » para o nome de domínio *mydomain.ovh*.

A seguir, poderá encontrar uma descrição completa das tags utilizadas para **o registo DMARC** da OVHcloud:

- **Version (v=)**: campo obrigatório que determina a versão do protocolo DMARC.

- **Regra para o domínio (p=)**: política a adotar pelo destinatário a pedido do proprietário do domínio remetente. A política aplica-se ao domínio inquirido e aos subdomínios, exceto se a marca de subdomínio **sp=** indicar instruções diferentes. Os valores possíveis são os seguintes:
    - *none* : o proprietário do domínio não pede nenhuma ação específica relativa à entrega das mensagens.

>     - *quarentena*: Se a verificação do mecanismo DMARC falhar, os destinatários devem considerar as mensagens de correio eletrónico como suspeitas. Dependendo das capacidades do servidor destinatário, isto pode significar « colocar na pasta de spam » e/ou « indicar como suspeito ».

Reviewer: ```suggestion
    - *quarentine*: Se a verificação do mecanismo DMARC falhar, os destinatários devem considerar as mensagens de correio eletrónico como suspeitas. Dependendo das capacidades do servidor destinatário, isto pode significar « colocar na pasta de spam » e/ou « indicar como suspeito ».
```

    - *reject* : rejeição dos e-mails que falham na verificação do mecanismo DMARC.

> [!warning]
>

> > A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. É aconselhável configurar `p=none` e efetuar uma análise dos relatórios de falha durante várias semanas, de forma a resolver eventuais anomalias. Passar para `p=quarantina` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.

Reviewer: ```suggestion
> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. É aconselhável configurar `p=none` e efetuar uma análise dos relatórios de falha durante várias semanas, de forma a resolver eventuais anomalias. Passar para `p=quarantine` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```

Depois de apresentar a zona DNS, clique no botão `Adicionar uma entrada`{.action} e, a seguir, em « Campos » `TXT`{.action}.

- **Subdomínio**: Esta entrada deve **obrigatoriamente começar por** `_dmarc`. Se você está aplicando seu DMARC ao conjunto do domínio, não insira nada além de `_dmarc` nesta caixa. Se definir o DMARC para um subdomínio associado ao seu domínio principal, adicione o subdomínio após `_dmarc`. Por exemplo, se for necessário aplicar o DMARC a um subdomínio *subdomain.mydomain.ovh*, é necessário introduzir `_dmarc.subdomain` na casa « subdomínio » para o nome de domínio *mydomain.ovh*

Abaixo, poderá encontrar a lista das tags utilizadas para criar um **registo TXT** com os parâmetros DMARC. Esta lista é complementar às tags mencionadas na secção anterior « [Registo DMARC](#dmarc-record) ».

- **adkim** (o padrão é `r`): indica o modo de alinhamento DKIM. Os valores são os seguintes:
    - `r`(relaxed) para o modo flexível: Os e-mails que falham na autenticação DKIM são marcados como « indesejados » pelo servidor destinatário.

>     - `s`(restrito) para o modo restrito: os e-mails que falham na autenticação DKIM são rejeitados pelo servidor destinatário.

Reviewer: ```suggestion
    - `s`(strict) para o modo restrito: os e-mails que falham na autenticação DKIM são rejeitados pelo servidor destinatário.
```

> - **ruf** - Endereços aos quais devem ser comunicadas as informações de erro específicas da mensagem, incluindo uma lista de texto simples separada por vírgulas Se esta tag estiver presente, o proprietário do domínio remetente pede aos destinatários que enviem relatórios de falhas detalhados sobre os e-mails que falham na avaliação DMARC de forma específica (ver a tag `fo` abaixo). O formato da mensagem a gerar deve seguir o formato especificado para a etiqueta `rf`. A menção « mailto:» deve preceder o destinatário de e-mail (exemplo: `mailto:address@exemplo.com`).

Reviewer: ```suggestion
- **ruf** - Endereços aos quais devem ser comunicadas as informações de erro específicas da mensagem, incluindo uma lista de texto simples separada por vírgulas Se esta tag estiver presente, o proprietário do domínio remetente pede aos destinatários que enviem relatórios de falhas detalhados sobre os e-mails que falham na avaliação DMARC de forma específica (ver a tag `fo` abaixo). O formato da mensagem a gerar deve seguir o formato especificado para a etiqueta `rf`. A menção « mailto:» deve preceder o destinatário de e-mail (exemplo: `mailto:address@example.com`).
```

     - **d** : gera um relatório de falhas DKIM se o mecanismo de autenticação DKIM falhar, independentemente do alinhamento.
     - **s** : gera um relatório de falhas SPF se o mecanismo de autenticação SPF falhar, independentemente do alinhamento.

- **rf** (texto simples separado por vírgulas, o valor predefinido é `afrf`) — Esta etiqueta indica o tipo de formato esperado para os relatórios que apresentam detalhes específicos sobre as falhas de autenticação das mensagens. Atualmente, é suportado apenas o `afrf` (Auth Failure Reporting Format).

- **ri** (inteiro não assinado de 32 bits para texto simples; o padrão é 86400) - Intervalo necessário, em segundos, entre as razões agregadas. Esta etiqueta especifica a frequência com que os destinatários do correio eletrónico geram relatórios agregados sobre os resultados da avaliação DMARC para o domínio em questão.

![dmarc](images/dns-dmarc-02.png){.thumbnail}

#### Exemplos de registo <a name="record-example"></a>

> [!warning]
>
> Nos nossos 2 exemplos, o parâmetro `p=`é utilizado na sua forma restritiva para ilustrar o comportamento de um serviço de e-mail nesse caso.
>

> > A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. Aconselhamos que configure `p=none` e efetue uma análise dos relatórios de falhas durante várias semanas, a fim de resolver eventuais anomalias. Passar para `p=quarantina` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.

Reviewer: ```suggestion
> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. Aconselhamos que configure `p=none` e efetue uma análise dos relatórios de falhas durante várias semanas, a fim de resolver eventuais anomalias. Passar para `p=quarantine` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```

##### Primeiro exemplo

Para ilustrar este primeiro exemplo, utilizámos o [registo DMARC](#dmarc-record) na zona DNS e aplicámos-lhe os seguintes parâmetros:

![dmarc](images/dns-dmarc-03.png){.thumbnail}

Obtemos o seguinte resultado:

```
"v=DMARC1;p=quarantine;pct=100;rua=mailto:report@mydomain.ovh;aspf=s;"
```

Todos os e-mails enviados (**pct=100**) são processados através dos mecanismos de autenticação SPF e/ou DKIM. Os e-mails que falharam no teste SPF são automaticamente rejeitados porque "**aspf=s**" (mecanismo SPF em modo restrito). Um relatório de erros sobre a falha dos mecanismos de autenticação SPF e/ou DKIM é enviado para o endereço `report@mydomain.ovh` (**rua=mailto:report@mydomain.ovh**).

##### Segundo exemplo

Neste segundo exemplo, usamos um [registo TXT](#txt-record) para usar etiquetas que não estão disponíveis através do [registo DMARC](#dmarc-record) simplificado.

![dmarc](images/dns-dmarc-04.png){.thumbnail}

Obtemos o seguinte resultado:

```
"v=DMARC1; p=quarantine; pct=100; ruf=mailto:report@mydomain.ovh; fo=0; adkim=r; aspf=s; adkim=r; ri=86400"
```

> - **p=quarantina**: As mensagens de correio eletrónico que não passem os testes DMARC são tratadas como « suspeitas ».

Reviewer: ```suggestion
- **p=quarantine**: As mensagens de correio eletrónico que não passem os testes DMARC são tratadas como « suspeitas ».
```
